snap/local/start-postgresql.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# For security measures, applications should not be run as sudo.
+export LOCPATH="${SNAP}"/usr/lib/locale
+export PGDATA=$SNAP_COMMON/pgsql/data
```

Comment:
To use the common shell style:
```suggestion
export PGDATA=${SNAP_COMMON}/pgsql/data
```